src/components/slide_in_panel.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Motion, spring } from 'react-motion';
+import classNames from 'classnames';
+
+export interface SlideInPanelProps {
+  /** Text to be displayed in the panel header */
+  headerTitle: string;
+  /** The panel will be open when true */
+  isOpen: boolean;
+  /** Called when the background overlay is clicked */
+  onOverlayClickHandler?: Function;
+  /** Width of panel. Include unit */
+  width?: string;
+  /** Show footer */
+  includeFooter?: boolean;
```

Comment:
should we include children here to explain how they are interpreted?

Reply:
I don't think you can do that